### PR TITLE
chore: Update ckb to v0.39

### PR DESCRIPTION
**Pull request by TheWaWaR**
* Update ckb to v0.39.0-rc2
* Add get_consensus and get_raw_tx_pool rpc methods

### CODE REVIEW

File: test/Cargo.toml [hunk 1]
```diff
@@ -14,11 +14,11 @@ env_logger = "0.6"
 toml = "0.5.0"
 serde_yaml = "0.8.9"
 ckb-sdk = { path = "../ckb-sdk" }
-ckb-types = { git = "https://github.com/nervosnetwork/ckb", tag = "v0.38.0-rc1" }
-ckb-app-config = { git = "https://github.com/nervosnetwork/ckb", tag = "v0.38.0-rc1" }
-ckb-chain-spec = { git = "https://github.com/nervosnetwork/ckb", tag = "v0.38.0-rc1" }
-ckb-crypto = { git = "https://github.com/nervosnetwork/ckb", tag = "v0.38.0-rc1", features = ["secp"] }
-ckb-hash = { git = "https://github.com/nervosnetwork/ckb", tag = "v0.38.0-rc1" }
+ckb-types = { git = "https://github.com/nervosnetwork/ckb", tag = "v0.39.0-rc1" }
+ckb-app-config = { git = "https://github.com/nervosnetwork/ckb", features=["with_sentry"], tag = "v0.39.0-rc1" }
```
Comment (doitian) [inline test/Cargo.toml:18]: Is there any problem without `with_sentry`?

Comment (TheWaWaR) [reply test/Cargo.toml:18]: If `with_sentry` feature is not enabled, serialize `CKBAppConfig` to string will not have `[sentry]` section, when deserialize the string back to `CKBAppConfig`  will have `missing sentry field` error.

Comment (TheWaWaR) [reply test/Cargo.toml:18]: @doitian 
Any questions?

Comment (doitian) [reply test/Cargo.toml:18]: If https://github.com/nervosnetwork/ckb/pull/2381 is merged this week, I'll cherry pick it into ckb v0.39.0

Comment (doitian) [reply test/Cargo.toml:18]: cc @quake 

It's weird why deserialization requires the `sentry` field. Is `ckb-app-config` also referenced indirectly by another crate which pass the `with_sentry` feature explicitly?

Comment (doitian) [reply test/Cargo.toml:18]: Try ckb v0.39.0-rc2

Comment (TheWaWaR) [reply test/Cargo.toml:18]: ok

**Review by doitian — APPROVED** [review]

**Review by driftluo — APPROVED** [review]

**Pull request merged**